.github/workflows/user-service-ci.yml

```
@@ -0,0 +1,94 @@
+name: HiveSpace UserService CI
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)